### PR TITLE
add CONTRIBUTING.md and CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by ranocha**
At least something like CONTRIBUTING.md is required for a JOSS paper.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to
+[David Kopriva](mailto:kopriva@math.fsu.edu)
+or any other of the principal developers responsible for enforcement listed in
+[AUTHORS.md](AUTHORS.md).
```
Comment (ranocha) [inline CODE_OF_CONDUCT.md:65]: We could also list other people (Andrew?) here directly

Comment (DavidAKopriva) [reply CODE_OF_CONDUCT.md:65]: Yes, Andrew should be added as a contact. As for me, I have to use the dkopriva@fsu.edu address now, since Oct. 1.

**Review by sloede — APPROVED** [review]
LGTM, thanks!

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (sloede) [reply CODE_OF_CONDUCT.md]: Fixed in [82481b5](https://github.com/trixi-framework/HOHQMesh/pull/110/commits/82481b55d083e4e886d1a4a35fdf7d7d3f515286).

**Pull request merged**